### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by betsy-lichtenberg**
I only included the javascript

### CODE REVIEW

File: convert-temp [hunk 1]
```diff
@@ -0,0 +1,16 @@
+$(function () {
```
Comment (larissam) [inline convert-temp]: Nice job hooking it up to the DOM! You actually only need to use ` $(function(){ ... })` once (wrap it around all your code) - what it does is make sure that the DOM has loaded before doing anything (important in bigger apps with a lot of things to load).

File: madlib-generator [hunk 2]
```diff
@@ -0,0 +1,22 @@
+let startupX = ['Uber', 'Google', 'Amazon', 'Apple', 'Facebook', 'Twitter'];
+let startupY = ['Slack', 'Trello', 'Tesla', 'Hyperloop', 'Harvest'];
+
+let random1 = Math.floor((Math.random() * startupX.length));
+let random2 = Math.floor((Math.random() * startupY.length));
+
+// console.log('A startup that is ' + startupX[random1] + ', but for ' + startupY[random2]);
+// s
+
+$( "#create" ).on( "click", function() {
```
Comment (larissam) [inline madlib-generator]: Nice job! 👍 Can you make this code generate a new startup every time the user presses `#create`?

File: 99-bottles [hunk 3]
```diff
@@ -0,0 +1,13 @@
+for (let totalBottles = 99; totalBottles > 0; totalBottles--) {
+  let bottleNoun = 'bottles';
```
Comment (larissam) [inline 99-bottles]: Nice variable name!

**Comment by betsy-lichtenberg** [comment]
Thanks for the feedback. My machine died, and I did the homework on a loaner chromebook, so I didn't have access to my regular tools--which is why I didn't submit it correctly.

**Pull request merged**